internal/profile/resource_profile.go

```diff
@@ -69,6 +69,8 @@ func (r ProfileResource) Schema(_ context.Context, _ resource.SchemaRequest, res
 
 			"project": schema.StringAttribute{
 				Optional: true,
+				Computed: true,
+				Default:  stringdefault.StaticString("default"),
```

Comment:
tiny nit, but we have a constant in the Incus API for that api.ProjectDefaultName.

Would be great to start using it within the Terraform provider too!

Reply:
Fixed.